ferretdb/ferretdb.go

```diff
@@ -34,6 +34,13 @@ type Config struct {
 	// Listen address; defaults to "127.0.0.1:27017".
 	ListenAddr string
 
+	// Listen unix domain socket path; defaults to "".
+	ListenSock string
+
+	// A flat to use a socket.
+	// If set to true while ListenSock is empty an 'abstract' socket might be used.
```

Comment:
I don't think we have a use-case for abstract sockets, let's not complicate it

Reply:
There's no complication here.
As far as I understand when we use empty string a socket with not file will be created, which is called abstract.

So this is just covers the case where we want to use unix sockets but we don't wanna give a path for it.

PS: It's actually a use case for integration tests

Reply:
I mean, there is not use-case for real users, and that complicates our code that we would have to maintain. Let's stick to "normal" filesystem-based Unix sockets